### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by jwaldrep**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ six==1.9.0
 sqlparse==0.1.15
 static3==0.6.1
 Werkzeug==0.10.4
+
```
Comment (benred42) [inline requirements.txt:22]: Is this extra line _really_ necessary?

Comment (jwaldrep) [inline requirements.txt:22]: **Yes**

Comment (benred42) [inline requirements.txt:22]: Alright, cool.

File: requirements.txt [hunk 2]
```diff
@@ -19,3 +19,4 @@ six==1.9.0
 sqlparse==0.1.15
 static3==0.6.1
 Werkzeug==0.10.4
```
Comment (jwaldrep) [inline requirements.txt:21]: We should take this out in production

**Pull request merged**